src/touchstone/compare.py

```diff
@@ -106,109 +151,152 @@ def main(args):
+                                                                index=index,
+                                                                input_dict=compare_uuid_dict) # noqa
+                        compute_aggs_set = \
+                            compute_aggs_set + database_instance._aggs_list
+                        compute_uuid_dict = \
+                            dict(mergedicts(compute_uuid_dict, _current_uuid_dict))
+                    compute_aggs_set = set(compute_aggs_set)
+                    compute_buckets = database_instance._bucket_list
+                    if print_csv:
+                        for key in compute_buckets:
+                            _compute_header += "{}, ".format(key)
+                        _compute_header += "key, uuid, value"
+                        if not printed_header:
+                            print(_compute_header)
+                            printed_header = True
+                    compare_dict(compute_uuid_dict, compute_aggs_set, _compute_value,
+                                 compute_buckets, args.uuid, _compute_header,
+                                 max_level=2 * len(compute_buckets), csv=print_csv)
+        if args.output == "json":
+            print(json.dumps(truth, indent=4))
+        elif args.output == "yaml":
+            print(yaml.dump(truth, allow_unicode=True))
+        elif args.output == "csv":
+            pass
+        else:
+            metadata += "\n{} End Metadata {}".format(("=" * 57), ("=" * 57))
+            print(metadata)
+
+    # If the query is for prometheus comparison
+    elif args.database == 'prometheus':
+        # If user provides the path for the configurations yaml file, data is retrieved from that
+        # at first
+        if args.prom_config is not None:
+            file_path = args.prom_config
+            file = open(file_path)
+            config = yaml.load(file, Loader=yaml.FullLoader)
+            query_list = config['query']
+            args.headers = config['headers']
+            disable_ssl = config['disable_ssl']
+            start_time_list = config['start_time_list']
+            end_time_list = config['end_time_list']
+            url_list = config['url']
+        # If no configuration file is provided, data is taken from command line arguments.
+        else:
+            start_time_list = args.start_time_list
+            end_time_list = args.end_time_list
+            query_list = [args.query_list]
+            url_list = args.conn_url
+            disable_ssl = False if args.disable_ssl == "False" else True
+
+        output = []
+        if args.headers is not None:
+            headers = {"Authorization": "bearer "+ args.headers[0]}
+        else:
+            headers = args.headers
+        # Iterate over each url provided to get aggregates for given query and time stamps.
+        for url in url_list:
+            prometheus_object = prometheus.Prometheus(query_list, start_time_list,
```

Comment:
replace with ```
            prometheus_object = databases.grab(args.database,query_list, start_time_list,
                                                      end_time_list, url=url, headers=headers,
                                                      disable_ssl=disable_ssl)
```

Reply:
@aakarshg made the changes. Although, there are some new changes in the prometheus-api-client pr. I'll update this pr accordingly as soon as that gets merged.